testing/apps/productpage/Dockerfile.sidecar

```diff
@@ -15,8 +15,8 @@
 FROM amalgam8/a8-sidecar:latest
 
 WORKDIR /opt/microservices
+COPY config.yaml /opt/microservices/
 COPY productpage /opt/microservices/
-COPY config.yaml /opt/microservices/config.yaml
-EXPOSE 9080
+EXPOSE 9080 12345
```

Comment:
why should productpage expose 12345 ? Its only for productpage app to Envoy. Nothing is going to talk to productpage over 12345